prod-refractr.yml

```diff
@@ -3,17 +3,15 @@
 # output from refractr.py
 default-domains:
   - prod.mozilla-redirects.xyz
-  - stage.mozilla-redirects.xyz
   - refractr.mozilla.org
-  - refractr.allizom.org
 
 refracts:
 
   # FIXME: this is a stage only tests
-- mozilla.com/: refractr1.allizom.org
+- mozilla.com/: refractr1.mozilla.org
 
   # FIXME: this is a stage only tests
```

Comment:
maybe want to change this comment if it's the prod domain?

Reply:
Fixed in c98caf9ece1e102a0515d0ec2b499209d45472d9